lib/src/Pos/Node/API.hs

```diff
@@ -547,47 +551,130 @@ instance ToSchema (V1 Version) where
         pure $ NamedSchema (Just "V1Version") $ mempty
             & type_ .~ SwaggerString
 
+
+newtype SecurityParameter = SecurityParameter Int
+    deriving (Show, Eq, Generic, ToJSON, FromJSON, Arbitrary)
+
+instance Buildable SecurityParameter where
+    build (SecurityParameter i) = bprint shown i
+
+instance ToSchema SecurityParameter where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "SecurityParameter") $ mempty
+            & type_ .~ SwaggerString
+
+
+instance ToSchema (V1 Core.SlotId) where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "Core.SlotId") $ mempty
+            & type_ .~ SwaggerString
+
+deriving instance ToJSON (V1 Core.SlotId)
+deriving instance FromJSON (V1 Core.SlotId)
+
+instance Arbitrary (V1 Core.SlotId) where
+    arbitrary = fmap V1 arbitrary
```

Comment:
> Do you actually need those instances? If it's about the Example type-class (which default to using an Arbitrary instance if any), you better go defining an example by hand.

— @KtorZ (https://github.com/input-output-hk/cardano-sl/pull/3971/files#r242542685)

Reply:
Think they're only used for the `Example` for `NodeSettings`. If so: are you sure?

Reply:
Try to remove them, look who's complaining :upside_down_face:

Reply:
Not possible. `Arbitrary` is a superclass of `Example`.
https://github.com/input-output-hk/cardano-sl/blob/5ecc6f58e6b4e0a92bbca696824f027387db6d86/lib/src/Pos/Util/Example.hs#L18-L20

We use `Example` here: (Which I guess could be changed, but that feel out of scope of this PR) https://github.com/input-output-hk/cardano-sl/blob/5ecc6f58e6b4e0a92bbca696824f027387db6d86/lib/src/Pos/Node/API.hs#L65

Reply:
Ah! True we did constrain every `Example` with `Arbitrary`. Sad. Let's not bother with that in this ticket then.